Scripts/Core/EditorVR.Menus.cs

```diff
@@ -6,15 +6,14 @@
 using UnityEditor.Experimental.EditorVR.Modules;
 using UnityEditor.Experimental.EditorVR.Utilities;
 using UnityEngine;
-using UnityEngine.InputNew;
 
 namespace UnityEditor.Experimental.EditorVR.Core
 {
 	partial class EditorVR
 	{
 		const float k_MainMenuAutoHideDelay = 0.125f;
 		const float k_MainMenuAutoShowDelay = 0.25f;
-		class Menus : Nested, IInterfaceConnector, ILateBindInterfaceMethods<Tools>
+		class Menus : Nested, IInterfaceConnector, ILateBindInterfaceMethods<Tools>, IConnectInterfaces
```

Comment:
Behavior of this seems slightly different now - previously you could add/remove a mainMenu with a null rayorigin - now code stops that from happening.  Was anything relying on that behavior before?

Reply:
I'm not sure this is any different. You can't add null keys to dictionaries. We should probably have been doing that null check beforehand. In fact, I'm not 100% sure it's required now, since we only call ConnectInterfaces on menus with ray origins.

I was careful to look for all usages of ConnectInterfaces to ensure that callers that didn't provide a ray origin weren't affected.

Reply:
Sounds good